licenses.yaml

```diff
@@ -2161,6 +2180,16 @@ libraries:
 
 ---
 
+name: Apache Hadoop
+license_category: binary
+module: extensions/druid-orc-extensions
+license_name: Apache License version 2.0
+version: 2.2.0
```

Comment:
This doesn't look like the standard Hadoop version we use (2.8.3, right now). Is there a reason for that?

Reply:
`mvn dependency:tree` shows that it's being pulled in by the `org.apache.orc:orc-core` dependency:
```
[INFO] +- org.apache.orc:orc-core:jar:1.5.5:compile
[INFO] |  +- org.apache.orc:orc-shims:jar:1.5.5:compile
[INFO] |  +- com.google.protobuf:protobuf-java:jar:3.1.0:compile
[INFO] |  +- io.airlift:aircompressor:jar:0.10:compile
[INFO] |  +- javax.xml.bind:jaxb-api:jar:2.3.1:compile
[INFO] |  |  \- javax.activation:javax.activation-api:jar:1.2.0:compile
[INFO] |  \- org.apache.hadoop:hadoop-hdfs:jar:2.2.0:compile
```